### PR TITLE
[SDL 0180] Broaden Choice Uniqueness

### PR DESCRIPTION
**Pull request by iCollin**
ATF Test Scripts to check https://github.com/smartdevicelink/sdl_core/issues/2346

This PR is **ready** for review.

### Summary
add tests for add command and CICS with duplicate menuNames

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test_scripts/API/SameMenuName/002_CreateInteractionChoiceSet.lua [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/Smoke/commonSmoke')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 5
+config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 0
```
Comment (jacobkeeler) [inline test_scripts/API/SameMenuName/002_CreateInteractionChoiceSet.lua:16]: Is this needed?

**Review by jacobkeeler — CHANGES_REQUESTED** [review]

File: test_scripts/API/SameMenuName/002_CreateInteractionChoiceSet.lua (hunk 1 above)
Comment (iCollin) [reply test_scripts/API/SameMenuName/002_CreateInteractionChoiceSet.lua]: Nope, removed in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2501/commits/01e13168961d10a402f672c27fc818280773889a

File: test_scripts/API/SameMenuName/001_AddCommand.lua [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    cmdID = 500,
+    menuParams = menuParams
+}
+
+--[[ Local Functions ]]
+local function addCommand(pParams)
+  local cid = common.getMobileSession():SendRPC("AddCommand", pParams)
+
+  local hmiRequestParams = { 
+    cmdID = pParams.cmdID,
```
[the remark this answers is not in the document]
Comment (iCollin) [reply test_scripts/API/SameMenuName/001_AddCommand.lua:36]: this suggestion has been committed, not sure why it still appears here.

[the remark this answers is not in the document]
Comment (iCollin) [reply test_scripts/API/SameMenuName/001_AddCommand.lua:36]: this suggestion was applied in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2501/commits/bc450bd2349375f4e58af4f7fd5b5fd0bf2b9c47

**Review by jacobkeeler — APPROVED** [review]

**Review by jacobkeeler — CHANGES_REQUESTED** [review]

**Review by jacobkeeler — APPROVED** [review]
